logic.js

```diff
@@ -0,0 +1,139 @@
+  // Initialize Firebase
+  var config = {
+    apiKey: "AIzaSyAH498q5xfRdRITk_-cg4OlD50_4zRz5SU",
+    authDomain: "jproject-f5600.firebaseapp.com",
+    databaseURL: "https://jproject-f5600.firebaseio.com",
+    projectId: "jproject-f5600",
+    storageBucket: "jproject-f5600.appspot.com",
+    messagingSenderId: "410141371595"
+  };
+  firebase.initializeApp(config);
+
+  // Create a variable to reference the database
+  var database = firebase.database();
+
+  var trainName = "";
+  var destination = "";
+  var firstTrainTime = "";
+  var frequency = 0;
+  var currentTime;
+
+
+  var databaseRef = database.ref('trainScheduler');
+  console.log("database: "+ database)
+  console.log("databaseRef: "+ databaseRef);
+
+  runTime();
+  displayTime();
+  runPanels();
```

Comment:
You should run these 3 functions in a 
```javascript 
$(document).ready
``` 
doing this will limit the number of bugs that can pop up otherwise.